src/Tizen.NUI.Components/Controls/RecyclerView/ItemSelectionMode.cs

```diff
@@ -15,29 +15,39 @@
  */
 
 using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Tizen.NUI.Components
 {
     /// <summary>
     /// Selection mode of CollecitonView.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
+    [SuppressMessage("Microsoft.Naming",
+                     "CA1720: Identifiers should not contain type names",
+                     Justification = "Single is the member of enum ItemSelectionMode. there are no possible danger to miss using Single Identifiers.")]
     public enum ItemSelectionMode
     {
         /// <summary>
         /// None of item can be selected.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
         None,
         /// <summary>
         /// Single selection. select item exclusively so previous selected item will be unselected.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        SingleSelection,
+        /// <since_tizen> 9 </since_tizen>
+        Single,
+        /// <summary>
+        /// Single selection always. select item exclusively but selection is always exist after being selected.
```

Comment:
lower letter after .
"is always exist" -> "always exists", but what you really mean - it's not possible to unselect all, so after user selects an item, there is always exactly one item selected